static/app/components/keyValueData/index.tsx

```diff
@@ -82,7 +89,12 @@ export function Content({
         {hasSuffix && (
           <div>
             {hasErrors && <AnnotatedTextErrors errors={errors} />}
-            {actionButton && <ActionButtonWrapper>{actionButton}</ActionButtonWrapper>}
+            {actionButton &&
+              (actionButtonAlwaysVisible ? (
+                actionButton
+              ) : (
+                <ActionButtonWrapper>{actionButton}</ActionButtonWrapper>
```

Comment:
Why do we wrap it in one occasion, and not in the other?

If I understand this correctly, `actionButtonAlwaysVisible` is set to true when there is custom metric available for the span attribute, and false when we want to offer the user possibility to create a new one. Can you maybe rename the variable or add a comment explaining what does the check `actionButtonAlwaysVisible ?` do?

Reply:
sure thing!